src/js/view/controls/components/menu/settings-menu.js

```diff
@@ -326,6 +329,8 @@ class SettingsMenu extends Menu {
         // If closed by keypress, focus appropriate element.
```

Comment:
If the `aria-exanded` SHOULD stay on line 316, then should it be removed in this close?

Reply:
whoops did not see this the first time around, but yes! Sorry for the oversight. I did realize that the settings menu `aria-expanded` was not being set to false when collapsed so I will push that up too!